CHANGELOG.md

```diff
@@ -1,3 +1,55 @@
+### citus v12.1.0 (September 12, 2023) ###
+
+* Adds support for PostgreSQL 16.0 (#7173)
+
+* Add `citus_schema_move()` function which moves tables within a
+  distributed schema to another node (#7180)
+
+* Adds `citus_pause_node()` UDF that allows pausing the node with given id
```

Comment:
ops, the name of the UDF is citus_pause_node_within_txn, we should fix the changelog: https://github.com/citusdata/citus/pull/7089/files#diff-3b79898c97cb0fb3169790554076d0c3ced79f6c9e0dfc2b37103fd7f0aba8ffR1383